tests/functional/ionIonBeam/ion_ion_beam1d.py

```diff
@@ -106,14 +111,14 @@ def yaebx(x, a, b):
     return a*np.exp(np.multiply(b, x))
 
 
-def growth_b_right_hand(run_path):
+def growth_b_right_hand(run_path,d):
+
     file = os.path.join(run_path, "EM_B.h5")
     times = get_times_from_h5(file)
-
+    dt = times[1]-times[0]
```

Comment:
this only works because `times` is coming from an `arange`. I would rather prefer `dt_stamps = 0.1` and use it at line 95 and 118